lib/toggl/TogglService.ts

```diff
@@ -79,6 +79,13 @@ export default class TogglService {
     this._statusBarItem = this._plugin.addStatusBarItem();
     this._statusBarItem = this._plugin.addStatusBarItem();
     this._statusBarItem.setText("Connecting to Toggl...");
+
+    this._plugin.registerDomEvent(
+      this._statusBarItem, "click", () => {
+        new Notice('Reconnecting to Toggl...')
+        this.setToken(this._plugin.settings.apiToken)
+      }
+    )
```

Comment:
Great QoL improvement. One addition I would add is to set the API status to `ApiStatus.UNTESTED` before resetting the API client, either before calling `this.setToken` or at the top of `this.setToken`. I'll add that change in the release.